forge/ee/routes/httpTokens/index.js

```diff
@@ -10,7 +10,7 @@ module.exports = async function (app) {
                         return
                     }
                     const teamType = await request.project.Team.getTeamType()
-                    if (!teamType.getFeatureProperty('ha', true)) {
+                    if (!teamType.getFeatureProperty('teamHttpSecurity', true)) {
```

Comment:
This is defaulting to `true` - so anytime that hasn't explicitly enabled this feature will have this feature enabled... probably not the desired result.

```suggestion
                    if (!teamType.getFeatureProperty('teamHttpSecurity', false)) {
```

This will, in turn, require the tests to be updated as well - as they benefit from the 'default to enabled' behaviour currently. They will need to explicitly enable the feature on the team type before running the test.